addon/utils/display-type.js

```diff
@@ -0,0 +1,155 @@
+import { assert } from '@ember/debug';
+
+// Basic fields
+import BestuursorgaanSelectorEditComponent from '@lblod/ember-submission-form-fields/components/custom-submission-form-fields/bestuursorgaan-selector/edit';
```

Comment:
Huge list of imports, but once we deduplicate the components it will be halved and it will be further reduced once we implement the component registration functionality and the custom tables can be extracted.